src/components/LuxInputText.vue

```diff
@@ -388,7 +388,6 @@ $color-placeholder: tint(rgb(149, 156, 167), 50%);
         -moz-osx-font-smoothing: grayscale;
         opacity: 1;
       }
-      &:hover,
```

Comment:
On hover in light theme we would want
&:hover {
text-decoration-color: var(--color-princeton-orange-on-white);
}